src/Components/Message/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export class Message extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {'msg': this.props.text}
+    }
+
+    componentWillReceiveProps(nextProps) {
+        this.setState({'msg':nextProps.text})
```

Comment:
```suggestion
        this.setState({'msg': nextProps.text})
```
being very picky, I think this has a space (as above on line 6)